api/trace/api.go

```diff
@@ -31,6 +31,9 @@ type TracerProvider interface {
 	// instrumentationName is empty, then a implementation defined default
 	// name will be used instead.
 	Tracer(instrumentationName string, opts ...TracerOption) Tracer
+
+	// Shutdown closes the provider's resources such as the span processors.
+	Shutdown()
```

Comment:
I would like to get resolution on https://github.com/open-telemetry/opentelemetry-specification/issues/1073 before we merge this change to the API. Looking at other implementations this is not something they included in the API (https://github.com/open-telemetry/opentelemetry-java/blob/78fd4d1c8f4c4f85282b124db92401eb1feb28df/api/src/main/java/io/opentelemetry/trace/TracerProvider.java#L20-L43, https://github.com/open-telemetry/opentelemetry-python/blob/0e852ea393aa042d429657b9d22208eb33b52d67/opentelemetry-api/src/opentelemetry/trace/__init__.py#L172-L178, https://github.com/open-telemetry/opentelemetry-js/blob/ae1185dbcdc41d666f7e9f47474ea8996dbc8968/packages/opentelemetry-api/src/trace/tracer_provider.ts#L22-L35), rather just the SDK (https://github.com/open-telemetry/opentelemetry-java/blob/78fd4d1c8f4c4f85282b124db92401eb1feb28df/sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/TracerSdkManagement.java#L58, https://github.com/open-telemetry/opentelemetry-python/blob/0e852ea393aa042d429657b9d22208eb33b52d67/opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py#L833, https://github.com/open-telemetry/opentelemetry-js/blob/ae1185dbcdc41d666f7e9f47474ea8996dbc8968/packages/opentelemetry-tracing/src/BasicTracerProvider.ts#L103-L105).

If we want to reduce the scope to just the SDK and unblock this from the specification issue we could follow the same by only implementing at the SDK level here.

Reply:
The solution from https://github.com/open-telemetry/opentelemetry-specification/issues/1073#issuecomment-707442967, we should add `Shutdown` to `TracerProvider` SDK but leave the `TracerProvider` interface unchanged.

Reply:
@XSAM , thanks for the update. If I understand you correctly, the PR shall be updated:

1. Remove the `Shutdown`  method in  the `TracerProvider interface {` in the `./trace.go`
2. Add the `shutdown` method to `TracerProvider struct ` in `./sdk/trace/provider.go`

Is my understanding correct? Thanks.